src/cinder/gl/draw.cpp

```diff
@@ -391,34 +391,6 @@ void draw( const PolyLine2 &polyLine )
 	ctx->popVao();
 }
 
-void draw( const PolyLine3 &polyLine )
-{
-	auto ctx = context();
-	const GlslProg* curGlslProg = ctx->getGlslProg();
-	if( ! curGlslProg ) {
-		CI_LOG_E( "No GLSL program bound" );
-		return;
-	}
-	
-	const vector<vec3> &points = polyLine.getPoints();
-	VboRef arrayVbo = ctx->getDefaultArrayVbo( sizeof(vec3) * points.size() );
-	arrayVbo->bufferSubData( 0, sizeof(vec3) * points.size(), points.data() );
-
-	ctx->pushVao();
-	ctx->getDefaultVao()->replacementBindBegin();
-	ScopedBuffer bufferBindScp( arrayVbo );
-	int posLoc = curGlslProg->getAttribSemanticLocation( geom::Attrib::POSITION );
-	if( posLoc >= 0 ) {
-		enableVertexAttribArray( posLoc );
-		vertexAttribPointer( posLoc, 3, GL_FLOAT, GL_FALSE, 0, (const GLvoid*)nullptr );
-	}
-
-	ctx->getDefaultVao()->replacementBindEnd();
-	ctx->setDefaultShaderVars();
-	ctx->drawArrays( ( polyLine.isClosed() ) ? GL_LINE_LOOP : GL_LINE_STRIP, 0, (GLsizei)points.size() );
```

Comment:
Hm, might also require an extra argument to determine if it is `isClosed` or not (why not add a default while there).